### ISSUE TITLE
XDEBUG SUPPORT! How does this not exist yet?

### ISSUE THREAD

**Issue by tripflex** [pantheon-systems/localdev-issues#40]
This should be basic developer stuff -- without XDebug -- i don't know how anybody actually debugs a PHP application, this is essential and not having this feature makes localdev for the most part, completely useless to any real developer.

**Comment by sambasile** [comment]
+1 to this.  Having this feature would make the Localdev product a well-rounded environment.

**Comment by taupecat** [comment]
+1 Xdebug please!

**Comment by tripflex** [comment]
Hello Pantheon? A least some kind of response would be nice -- how do you expect actual developers to develop without XDEBUG or some kind of debugger support??

https://docs.lando.dev/config/pantheon.html#using-xdebug

Seems pretty basic stuff

**Comment by taz77** [comment]
Just ran into this, shocked XDebug does not have an easy path of usage.

**Comment by tripflex** [comment]
@taz77 don't waste your time with localdev, setup lando and use that instead.

**Comment by taz77** [comment]
Yup I already went that route. I deleted the Localdev app and will just us Lando.

**Comment by taupecat** [comment]
Finally took the dive into Lando and now I'm all-in. And I have XDebug working the way I want. Thanks for the tip, @tripflex.

**Comment by mfs-mc** [comment]
It's been over a year, is Xdebug still MIA? If so, then I presume Lando is the way to go. Anybody?
